### PR TITLE
fix: reduce package size and use path aliases

### PR DESCRIPTION
**Pull request by imbhargav5**

### CODE REVIEW

File: packages/rooks/.npmignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+*
+!dist/**/*
+dist/esm/__tests__
```
Comment (imbhargav5) [inline packages/rooks/.npmignore:3]: This file can probably be deleted.

File: .npmignore [hunk 2]
```diff
@@ -1,7 +1,8 @@
 *
 !dist/**/*
-dist/esm/__tests__/**/*
-dist/cjs/__tests__/**/*
+dist/esm/__tests__
```
Comment (imbhargav5) [inline .npmignore:3]: This file can probably be deleted now.

**Comment by imbhargav5** [comment]
@qqpann Thoughts on this PR?

File: .npmignore (hunk 2 above)
Comment (qqpann) [reply .npmignore:3]: I see you have two `.npmignore` in the root and packages/rooks.
I guess we only need one of them

I checked with
```sh
# in packages/rooks
npm pack --dry-run
```
and removing either one of them seems to work fine.

Comment (qqpann) [reply .npmignore:3]: Is that what you meant with
> This file can probably be deleted now.

?

Comment (qqpann) [reply .npmignore:3]: @imbhargav5 What is your workflow to publish rooks to npm?
Do you use `npm publish` or let some tools handle it?

**Review by qqpann — APPROVED** [review]
Path aliases look pretty!

There is one remaining in the spec template FYI.

**Pull request merged**